SeleniumTestXUnit/Tests/Features/User/Logout.feature

```diff
@@ -0,0 +1,12 @@
+Feature: User Logout
+    As a logged in user, the user should be able to Logout
+
+    Background:
+        Given the user is logged in
+
+    @Draft
+    @Smoke
+    Scenario: Logout off the site succesfully
+        When the user clicks the logout button
+        Then the user should be logged out from the site
```

Comment:
Looks like this validation is not really needed, the following validation (that the login view is displayed) should be enough to assert that the user has logged out.